### DISCUSSION TITLE
Mount is sometime not update new file from v2.49 to v2.58

### DISCUSSION THREAD

**Discussion by eltphat** [seaweedfs/seaweedfs#2197] [Q&A]
My installation: 3 nodes
1 master
weed master -defaultReplication=000 -mdir=/root/seaweedfs/master -ip=10.10.4.3
2 data + filer
weed volume -dir=/root/seaweedfs/data -max=0 -mserver=10.10.4.3:9333 -ip=10.10.4.5 -publicUrl=xxx.xxx.xxx.xxx:8080
weed volume -dir=/root/seaweedfs/data -max=0 -mserver=10.10.4.3:9333 -ip=10.10.4.6 -publicUrl=xxx.xxx.xxx.xxx:8080
weed filer -master=10.10.4.3:9333 -ip=10.10.4.5 -peers=10.10.4.5:8888,10.10.4.6:8888
client
weed mount -filer=10.10.4.5:8888 -dir=/data -filer.path=/ -volumeServerAccess=filerProxy

It's ok with v2.48 but with v2.49 - v2.58 is not. When I update files from client-1, but in client-2 don't update that files. It only update when I restart `weed mount`

Anyone get on this like me. Please help. Thanks

**Comment by chrislusf** [comment]
Not clear:
* what is the filer store
* how the 2 mounts connect to the 2 filers.

**Reply by eltphat** [reply]
Here is filer config:
```
[leveldb2]
enabled = true
dir = "/root/seaweedfs/filer"
```

I use this cli to connect to filer in private network
`weed mount -filer=10.10.4.5:8888 -dir=/data -filer.path=/ -volumeServerAccess=filerProxy`

**Reply by chrislusf** [reply]
how the other mount is connected?

**Reply by eltphat** [reply]
All client used above command
For external I use
`weed mount -filer=public-ip:8888 -dir=/data -filer.path=/ -volumeServerAccess=filerProxy`

**Reply by chrislusf** [reply]
what is the exact command line?

**Reply by eltphat** [reply]
Above command line is exactly I using, public-ip is my public ip

**Reply by eltphat** [reply]
it worked perfectly for v2.48 even earlier version.
From v2.49 to v2.58, sometimes it happen like I said above, I have to restart it

**Reply by chrislusf** [reply]
it's vague just with words. Could you make a reproducible case? maybe with a docker compose file?
